### ISSUE TITLE
What's the version of ms-swift you use?

### ISSUE THREAD

**Issue by Go2Heart** [mc-lan/Text4Seg#7]
I have encounter some version issue with the latest swift, like the arg `model_type` changing to `model`. Can you provide your swift version?

**Comment by mc-lan** [comment]
We used swift 2.x.

**Comment by Go2Heart** [comment]
Thanks for your replying. I am still facing problems in swift 2.x. Can you give me your full pip list for reproducing the table-1,2,3.

**Comment by mc-lan** [comment]
We have added [my_requirements.txt](https://github.com/mc-lan/Text4Seg/blob/76534478557d5e19efaa88e1cc63520795701aaa/ms-swift/my_requirements.txt). Hopefully, this resolves your issue. 
Are you trying to reproduce the results using our checkpoints, or do you plan to re-train the models?
We recommend using the latest ms-swift3.x. With minimal modifications to `data/create_refcoco_***.py` to generate the required JSON files, followed by SFT, you should be able to achieve good results.

**Comment by Go2Heart** [comment]
I am re-training the model, from llava1_5_hf. Could you specify the modifications needed for create_refcoco_llava.py?

**Comment by Go2Heart** [comment]
I believe the main problems here is that swift 3.x has different instruction args, which doesn't fit your training script.

**Comment by mc-lan** [comment]
You may refer [here](https://swift.readthedocs.io/zh-cn/latest/Customization/%E8%87%AA%E5%AE%9A%E4%B9%89%E6%95%B0%E6%8D%AE%E9%9B%86.html).

{"messages": [{"role": "user", "content": "<image>Might you segment this image?"}, {"role": "assistant", "content": "Sure, the segmentation mask is: <seg> semantic descriptors</seg>"}], "images": ["/xxx/x.jpg"]}

**Comment by Go2Heart** [comment]
Thanks for your info. I have begun my training. What's your ETA time for training the refcoco? I am at 40hours with 4 H100.

**Comment by mc-lan** [comment]
For llava1_5-7B, around 17h with 8 A800 40G.

**Comment by mc-lan** [comment]
{"loss": 0.01048364, "grad_norm": 0.04835183, "learning_rate": 0.0, "memory(GiB)": 35.49, "train_speed(iter/s)": 0.563091, "epoch": 5.0, "global_step/max_steps": "33930/33930", "percentage": "100.00%", "elapsed_time": "16h 44m 11s", "remaining_time": "0s"}

To clarify, we finally utilized **32**x A800 GPUs to accelerate model training.
H100 is really fast!

**Comment by Go2Heart** [comment]
Thanks for your prompt feedback. My questions have been resolved.
